### PR TITLE
[11.6] Added addShare method

### PR DESCRIPTION
**Pull request by peterjaap**

### CODE REVIEW

File: src/Api/Groups.php [hunk 1]
```diff
@@ -205,6 +205,35 @@ public function removeMember($group_id, int $user_id)
     {
         return $this->delete('groups/'.self::encodePath($group_id).'/members/'.self::encodePath($user_id));
     }
+    
+    /**
+     * @param int|string $group_id
+     * @param array      $parameters
```
Comment (GrahamCampbell) [inline src/Api/Groups.php]: Please add phpdoc for this. I cannot accept this PR without this. Please see the other methods for an example.

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]
Thanks for the PR. I've left some feedback inline.

File: src/Api/Groups.php (hunk 1 above)
Comment (peterjaap) [reply src/Api/Groups.php]: Done

**Comment by GrahamCampbell** [comment]
Thanks. :)

**Pull request merged**